modules/control/README.md

```diff
@@ -3,6 +3,11 @@
 ## Introduction
 Based on the planning trajectory and the car's current status, the Control module uses different control algorithms to generate a comfortable driving experience. The Control module can work both in normal and navigation modes.
 
+Apollo 5.0 Control has following new features:
+ * **Low Speed / Reverse Driving**: The control module now enables more accurate low-speed forward and backward driving, and therefore supports the valet parking, pull-over and other practical driving scenarios.
```

Comment:
The control module ensures accurate low-speed forward and backward driving, which supports complex and practical driving scenarios like valet parking and pull-over to name a few.

Reply:
Fixed per comment. Thank you.